app/views/ppm_estimator/_estimate_table.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="ppm-estimate-table-title">
+  Your PPM Incentive Estimate:<a id="ppm-edit-details" onclick="onEditDetails()" href="javascript:void(0);">Edit Details</a>
```

Comment:
i've generally always avoided any inline scripting, not sure if we have any conventions around that for move.mil

Reply:
@jgarber623-gov Any suggestions for how I should do this differently?

Reply:
Couple of ideas:

1. This element should be a `<button id="ppm-edit-details">`.
2. You can attach an event handler in the `ppm-estimator.js` JavaScript file:

```js
$('#ppm-edit-details').on('click', function(event) {
  event.preventDefault();

  onEditDetails(); // <= or whatever other code you want to execute on click…
});
```